.github/workflows/e2e-visual.yml

```diff
@@ -0,0 +1,88 @@
+name: E2E Visual
+on: workflow_dispatch
+  # pull_request:
```

Comment:
Are these lines intentionally left commented out?

Reply:
Yes intentional as there is an issue with Safari so kept it as a manual trigger for now.